startup_assistance/indico_startup_assistance/tasks.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+from datetime import date
+
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.core.notifications import make_email, send_email
+from indico.core.plugins import get_plugin_template_module
+from indico.modules.rb.models.reservation_occurrences import ReservationOccurrence
+from indico.modules.rb.models.reservations import Reservation
+
+from indico_startup_assistance.plugin import StartupAssistanceRequestPlugin
+
+
+def _send_email(recipients, template):
+    email = make_email(from_address='noreply-indico-team@cern.ch', to_list=recipients, template=template, html=True)
+    send_email(email, module='Indico Startup Assistance CERN')
```

Comment:
no need to set a module, it's only used when sending an email within an event (it's used for the event log)